llm_on_ray/inference/serve.py

```diff
@@ -152,6 +152,12 @@ def main(argv=None):
     parser.add_argument(
         "--max_batch_size", default=None, type=int, help="The max batch size for dynamic batching."
     )
+    parser.add_argument(
+        "--openai_route_prefix",
+        default=None,
+        type=str,
+        help="Whether to use default '/' route prefix or deploy at new route prefix.",
+    )
```

Comment:
Let's set the default value as '/' here. As ray will check the route_prefix value https://github.com/ray-project/ray/blob/a0deb6b40cb4149d9a59db6df1bedb7402b09056/python/ray/serve/api.py#L473-L476 , we can add this instruction `The openai_route_prefix must start with a forward slash ('/')` in help string to prompt users.

Reply:
Thanks for the input, implemented this in the new commit